packages/layout/views/FileView.tsx

```diff
@@ -0,0 +1,32 @@
+import { flattenToAppURL } from '@plone/helpers';
+import { useRouteLoaderData } from 'react-router';
+import type { RootLoader } from 'seven/app/root';
+import { useTranslation } from 'react-i18next';
+import { Container } from '@plone/components';
+
+export default function FileView() {
+  const rootData = useRouteLoaderData<RootLoader>('root');
+  const { t } = useTranslation();
+
+  if (!rootData) {
+    return null;
+  }
+
+  const { content } = rootData;
+
+  return (
+    <Container className="view-wrapper">
+      <h1 className="documentFirstHeading">{content.title}</h1>
+      {content.description && (
+        <p className="documentDescription">{content.description}</p>
+      )}
+      {content.file?.download && (
+        <a href={flattenToAppURL(content.file.download)}>
+          {content.file.filename
+            ? content.file.filename
+            : t('layout.FileView.download')}
+        </a>
```

Comment:
I recommend using the Link component from `@plone/components`  that takes care of client side rendering giving us a little perf boost. I would add the download attribute with the file name while you're there, as we did in the Image View: https://github.com/plone/volto/pull/7513

Final note, I forgot to tell everybody at the introduction, we are not using flattenToAppURL anymore. TLDR: we don't need that in Seven. The long story is that we are basically doing that for everything that we receive from the backend upfront, in the function that fetches the data.